sdks/java/io/pulsar/src/main/java/org/apache/beam/sdk/io/pulsar/PulsarIO.java

```diff
@@ -87,10 +83,6 @@ abstract Builder setExtractOutputTimestampFn(
       abstract Read build();
     }
 
-    public Read withAdminUrl(String url) {
```

Comment:
I don't think we can remove this due to backwards compatibility (seems like this connector not marked experimental).

Can this be addressed by deprecating ?

Reply:
+1 for deprecating the APIs saying that they are now unused.